railties/lib/rails/engine.rb

```diff
@@ -587,10 +587,6 @@ def load_seed
       config.eager_load_paths.freeze
     end
 
```

Comment:
I have to remove this intitializer as it is useless when `routes` is accessed directly

check the first commit for an attempt to keep it : https://github.com/rails/rails/pull/54916/commits/3346b19d0025a2048f975e7eec9ac9eac74ff908

I've tried the first commit on my rails app, and setting `exceptions_app` early on, or even in a `after_initialize` does not work